test_vm/tests/change_owner_test.rs

```diff
@@ -1,163 +1,267 @@
-use fil_actor_miner::{ChangeBeneficiaryParams, Method as MinerMethod};
+
+use fil_actor_miner::ChangeBeneficiaryParams;
 use fvm_ipld_blockstore::MemoryBlockstore;
-use fvm_shared::bigint::Zero;
+use fvm_shared::crypto::signature::SignatureType;
 use fvm_shared::econ::TokenAmount;
-use fvm_shared::error::ExitCode;
 use fvm_shared::sector::RegisteredSealProof;
-use test_vm::util::{
-    apply_code, change_beneficiary, change_owner_address, create_accounts, create_miner,
-    get_beneficiary,
-};
-use test_vm::VM;
+use fvm_shared::state::StateTreeVersion;
+use fvm_shared::version::NetworkVersion;
+use fvm_workbench_api::wrangler::ExecutionWrangler;
+use fvm_workbench_api::WorkbenchBuilder;
+use fvm_workbench_builtin_actors::genesis::{create_genesis_actors, GenesisSpec};
+use fvm_workbench_vm::builder::FvmBenchBuilder;
+use fvm_workbench_vm::externs::FakeExterns;
+use test_vm::bench::Benchmarker;
+use test_vm::util::{change_beneficiary_, change_owner_address_, create_accounts_, create_miner_};
+use test_vm::TestVM;
 
-#[test]
-fn change_owner_success() {
-    let store = MemoryBlockstore::new();
-    let mut v = VM::new_with_singletons(&store);
-    let addrs = create_accounts(&v, 3, TokenAmount::from_whole(10_000));
+enum ConcreteVM<'vm, 'bs> {
+    TestVM(&'vm TestVM<'bs>),
+    BenchmarkVM(&'vm Benchmarker<'vm>),
+}
+
+/// Sample test that can be injected with either a TestVM (rust execution, stack traceable etc) or a
+/// _BenchmarkVM (wasm execution, gas metering etc)
+fn change_owner_test(concrete_vm: ConcreteVM) {
+    let vm = match concrete_vm {
+        ConcreteVM::TestVM(tvm) => tvm.vm(),
+        ConcreteVM::BenchmarkVM(vm) => vm,
+    };
```

Comment:
I'm not aiming to have an alternative VM imported here at all – this has gone a bit farther than I think we should initially. This method should just take a `&dyn VM`